qiskit_addon_obp/utils/visualization.py

```diff
@@ -36,7 +36,7 @@
 from .metadata import OBPMetadata
 
 
-def plot_accumulated_error(metadata: OBPMetadata, axes: Axes) -> None:
+def plot_accumulated_error(metadata: OBPMetadata, axes: Axes, show_legend: bool = True) -> None:
```

Comment:
```suggestion
def plot_accumulated_error(metadata: OBPMetadata, axes: Axes, *, show_legend: bool = True) -> None:
```
This enforces that `show_legend` may only be used as a keyword-argument. For something like this, I consider this good practice because it avoids future API complications in case we want to add "higher priority" arguments to these function signatures which could become positional arguments.

Please apply this to the other functions accordingly. This may require another round of `tox -e style`.

Reply:
Great suggestion, done!